app/views/users/_unfollow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for current_user.active_relationships.find_by(followed_id: @user.id),
+    html: { method: :delete }, remote: true do |f| %>
```

Comment:
`current_user.active_relationships.find_by(followed_id: @user.id)`
Handle this in the controller/helper